executor-service/src/main/java/executor/service/facade/execution/ExecutionServiceImpl.java

```diff
@@ -1,16 +1,17 @@
 package executor.service.facade.execution;
 
-import executor.service.annotation.Autowired;
 import executor.service.holder.ScenarioQueueHolder;
 import executor.service.model.Scenario;
 import executor.service.service.executor.ScenarioExecutor;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.openqa.selenium.WebDriver;
-
 import java.util.concurrent.BlockingQueue;
 import java.util.function.Supplier;
+import org.springframework.stereotype.Service;
+import executor.service.annotation.Autowired;
```

Comment:
We can now use Autowired with springframework:
```import org.springframework.beans.factory.annotation.Autowired;```

Reply:
done, thanks!